x-pack/plugins/cases/common/api/connectors/index.ts

```diff
@@ -84,14 +84,22 @@ export const ConnectorTypeFieldsRt = rt.union([
   ConnectorSwimlaneTypeFieldsRt,
 ]);
 
+/**
+ * This type represents the connector's format when it is encoded within a user action.
+ */
+export const CaseUserActionConnectorRt = rt.intersection([
```

Comment:
This type just lacks the `id` field since that wont exist in the encoded data any longer.